README.md

```diff
@@ -48,10 +48,16 @@ tables = ["t3", t4]
 
 ## Rule
 
-By default, go-mysql-elasticsearch will use MySQL table name as the Elasticserach's index and type name, use MySQL table field name as the Elasticserach's field name.
-e.g, if a table named blog, the default index and type in Elasticserach are both named blog, if the table field named title,
+By default, go-mysql-elasticsearch will use MySQL table name converted to lowercase as the Elasticserach's index and type name, use MySQL table field name as the Elasticserach's field name.
+e.g, if a table named BLOG, the default index and type in Elasticserach are both named blog, if the table field named title,
 the default field name is also named title.
 
+
+If the actual table name is in uppercase and you get an error that like below, use the following [reference](https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_lower_case_table_names) to add the dbms setting.  
```

Comment:
no need this, if we user lowercase now, we must convert the table to lowercase when syncing too.

Reply:
so if the user has two tables Blog and blog, we can't sync and must raise an error when creating the rule.